athena-gcs/src/test/java/com/amazonaws/athena/connectors/gcs/filter/AthenaMarker.java

```diff
@@ -0,0 +1,50 @@
+/*-
+ * #%L
+ * athena-storage-api
+ * %%
+ * Copyright (C) 2019 - 2022 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connectors.gcs.filter;
+
+import com.amazonaws.athena.connector.lambda.data.Block;
+import com.amazonaws.athena.connector.lambda.domain.predicate.Marker;
+
+public class AthenaMarker extends Marker
```

Comment:
Is this class used anywhere other than inside a test?
If not, it should be deleted.
Same comment goes for any other class that is unnecessary.

Reply:
removed.